functional_tests/feature-flags.test.ts

```diff
@@ -178,7 +178,7 @@ describe('FunctionalTests / Feature Flags', () => {
             expect(getRequests(token)['/decide/']).toEqual([
                 // This is the initial call to the decide endpoint on PostHog init, with all info added from `loaded`.
                 {
-                    // $anon_distinct_id: 'anon-id', // no anonymous ID is sent because this was overridden on load
+                    $anon_distinct_id: 'anon-id',
```

Comment:
I'm not certain but I can't see any reason why we _wouldn't_ have wanted the anon id involved. The comment about it not being set because it was overridden is not true at all. It just wasn't set because decide.ts was the previous thing that triggered first and for some reason it didn't include it.